electricitymap/contrib/lib/models/events.py

```diff
@@ -430,6 +430,33 @@ def create(
                 },
             )
 
+    @staticmethod
+    def update(event: "Exchange", new_event: "Exchange") -> "Exchange":
```

Comment:
Not sure I fully understand why this would be usef by someone if they already created an Exchange.
Maybe this should be an internal method of the list rather?
I'm not sure someone would understand what this is for.

Reply:
The reason this is in here is because it vastly simplifies the logic in the lists and can be abstracted away (could even be possible to do in the top level event_list if all the events types have a update function).

But I'll prepend it weith `_`to mark it as private.